src/components/App.js

```diff
@@ -35,15 +49,14 @@ class App extends Component {
           <input type="search" name="keyword" placeholder="Enter topic or organisation" />
           <AddressFinder data={this.props} />
           <button type="submit">Search</button>
-        </form>
-        <div>
-          {this.props.results.length > 0 &&
-            <button className="btn-toggle" onClick={() => {
-              this.setState({ showMap: !this.state.showMap}); }
-            }>{this.state.showMap ? 'Show List' : 'Toggle Map'}</button>
+          {this.props.itemsLoading && false &&
+            <i className="loader"><img src="loading.gif" alt="loading..." /></i>
```

Comment:
@verbman is there any reason why we are using a gif here rather than font icon?

Reply:
I've removed those lines, we're still using the loading.gif but from css - my preference is for the gif being easier to maintain/implement